aws/lambda/pytorch-auto-revert/pytorch_auto_revert/workflow_checker.py

```diff
@@ -138,20 +146,42 @@ def restart_workflow(self, workflow_name: str, commit_sha: str) -> None:
         # Resolve workflow (exact display or file name)
         wf_ref = self.resolver.require(workflow_name)
 
+        # Check what inputs this workflow supports
+        input_support = self.resolver.get_input_support(workflow_name)
```

Comment:
be resilient here.

Accept that this function could fail (raise an exception, etc).

And if it does, just assume the workflow does not support inputs and move on. It is usually better to be suboptimal when things are unstable than to not do anything at all and fail.